cpp/benchmarks/common/generate_input.cu

```diff
@@ -206,42 +274,35 @@ struct random_value_fn<T, std::enable_if_t<!std::is_same_v<T, bool> && cudf::is_
   {
   }
 
-  T operator()(std::mt19937& engine)
+  auto operator()(thrust::minstd_rand& engine, unsigned size)
   {
     // Clamp the generated random value to the specified range
-    return std::max(std::min(dist(engine), upper_bound), lower_bound);
+    // return std::max(std::min(dist(engine), upper_bound), lower_bound);
```

Comment:
why is clamp not required here?

Reply:
updated `value_generator` with `std::clamp`.